src/reply-password/reply-password.c

```diff
@@ -35,7 +35,7 @@ static int send_on_socket(int fd, const char *socket_name, const void *packet, s
 }
 
 int main(int argc, char *argv[]) {
-        _cleanup_free_ char *packet = NULL;
+        _cleanup_string_free_erase_ char *packet = NULL;
```

Comment:
while this works I am not really liking this too much. In theory the datagrams should be able to carry multiple passwords, i.e. contain embedded NUL bytes. And we really should erase the whole thing in that case. Of course this tool doesn't actually allow replying with multiple passwords, but I still would feel more comfortable if we'd erase the full datagram, just in case it eventually gets updated to supported embedded NUl bytes, i.e. reporting multiple passwords.

I hope that makes some sense?

Reply:
OK. Now I adapt more simpler fix: initialize `length` by zero.